packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/pools/_components/poolsTableColumns.tsx

```diff
@@ -0,0 +1,140 @@
+import { useTranslation } from 'react-i18next';
+import { ColumnDef } from '@tanstack/react-table';
+
+import { MoreHorizontal } from 'lucide-react';
+import { SortableHeader } from '@/components/ui/data-table';
+import {
+  Tooltip,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@/components/ui/tooltip';
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+} from '@/components/ui/dropdown-menu';
+import { Button } from '@/components/ui/button';
+import { Span } from '@/components/typography';
+
+import { database } from '@/models/database';
+import { useServiceData } from '../../layout';
+import { GenericUser } from '@/api/databases/users';
+
+interface ConnectionPoolsTableColumnsProps {
+  databases: database.service.Database[];
+  users: GenericUser[];
+  onGetInformationClick: (
+    connectionPool: database.postgresql.ConnectionPool,
+  ) => void;
+  onEditClick: (connectionPool: database.postgresql.ConnectionPool) => void;
+  onDeleteClick: (connectionPool: database.postgresql.ConnectionPool) => void;
+}
+export const getColumns = ({
+  databases,
+  users,
```

Comment:
Avoid passing data to the columns definition, instead:
- declare a new type `ConnectionPoolWithData`
- in the new type, add fields for `database` and `user`
- in the listing page, create a new array of  ConnectionPoolWithData and map the data